bots-deployment-detector/package.json

```diff
@@ -28,7 +28,8 @@
     "test": "jest"
   },
   "dependencies": {
-    "forta-agent": "^0.1.32"
+    "forta-agent": "^0.1.32",
+    "forta-agent-tools": "^3.2.7"
```

Comment:
`forta-agent-tools` should be inside devDep because it's only used for testing for this bot.